src/main/webapp/i18n/de/assessment.json

```diff
@@ -9,7 +9,16 @@
             "feedbackHint": "Dieses Feedback ist mit einer Bewertungsanweisung verbunden. Du kannst zusätzliches Feedback für dieses Abgabeelement geben. Der/Die Studierende sieht das kombinierte Feedback während der Überprüfung.",
             "feedbackCommentPlaceholder": "Du kannst hier ein Feedback abgeben...",
             "additionalFeedbackCommentPlaceholder": "Du kannst hier zusätzliches Feedback abgeben...",
-            "automaticGenerated": "Automatisch erstelltes Feedback",
+            "suggestion": {
+                "suggested": "Feedback-Vorschlag (nicht akzeptiert)",
```

Comment:
I think between "not applied" and "nicht akzeptiert" there is some significant difference.

Reply:
Good catch! I chose to use "not accepted" in the English version.